launcher/ui/themes/ThemeManager.cpp

```diff
@@ -266,6 +266,17 @@ QString ThemeManager::addCatPack(std::unique_ptr<CatPack> catPack)
     return id;
 }
 
+void ThemeManager::removeCatPack(const QString& id) {
+    m_catPacks.erase(id);
+    if (APPLICATION->settings()->get("BackgroundCat").toString() == id) {
+        // set the selected cat to a random one of the remaining ones...
+        auto idx = (quint32) QRandomGenerator::global()->bounded((quint32) m_catPacks.size());
```

Comment:
Why do you need to cast it?
```suggestion
        auto idx = QRandomGenerator::global()->bounded(m_catPacks.size());
```

Reply:
Also here would be best to just take the first catpack from the list(do not randomize)

Reply:
It is casted bc. in my dev environment with qt6 it complains that the call matches multiple overloads. But even after the tried patch the CI complains... so propbably indeed the best to just reset it to the first one and scrap the idea of randomized cats entirely. Will add a commit.